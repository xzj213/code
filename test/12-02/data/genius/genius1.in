30
7 6
0.753 0.603 0.218 0.633 0.722 0.618 0.663 
0 1
1 2
0 3
2 4
3 5
5 6
9 8
0.040 0.118 0.619 0.036 0.748 0.662 0.492 0.059 0.700 
0 1
1 2
2 3
1 4
3 5
4 6
4 7
0 8
7 6
0.197 0.782 0.318 0.848 0.570 0.866 0.579 
0 1
1 2
0 3
3 4
3 5
3 6
7 6
0.234 0.666 0.095 0.473 0.707 0.862 0.798 
0 1
1 2
0 3
3 4
1 5
0 6
6 5
0.434 0.469 0.291 0.438 0.701 0.254 
0 1
1 2
1 3
1 4
1 5
5 4
0.790 0.653 0.291 0.935 0.303 
0 1
1 2
2 3
0 4
9 8
0.810 0.056 0.981 0.048 0.854 0.646 0.320 0.508 0.087 
0 1
0 2
1 3
1 4
4 5
1 6
3 7
2 8
8 7
0.211 0.856 0.988 0.254 0.697 0.604 0.763 0.855 
0 1
0 2
0 3
2 4
0 5
5 6
4 7
7 6
0.354 0.786 0.444 0.774 0.881 0.777 0.549 
0 1
1 2
1 3
3 4
2 5
3 6
6 5
0.663 0.662 0.010 0.888 0.459 0.783 
0 1
0 2
0 3
2 4
2 5
10 9
0.148 0.311 0.494 0.583 0.223 0.033 0.533 0.868 0.673 0.437 
0 1
0 2
2 3
3 4
4 5
5 6
6 7
5 8
5 9
6 5
0.127 0.744 0.800 0.294 0.390 0.649 
0 1
0 2
0 3
2 4
0 5
6 5
0.377 0.697 0.612 0.593 0.959 0.646 
0 1
0 2
2 3
3 4
0 5
10 9
0.203 0.318 0.819 0.540 0.365 0.619 0.732 0.263 0.442 0.232 
0 1
1 2
0 3
2 4
3 5
1 6
5 7
6 8
3 9
8 7
0.467 0.367 0.231 0.364 0.621 0.134 0.283 0.544 
0 1
0 2
2 3
2 4
4 5
1 6
5 7
6 5
0.068 0.412 0.041 0.850 0.790 0.303 
0 1
1 2
2 3
3 4
1 5
5 4
0.757 0.914 0.044 0.704 0.417 
0 1
0 2
2 3
0 4
5 4
0.547 0.257 0.221 0.147 0.054 
0 1
1 2
0 3
0 4
5 4
0.605 0.119 0.513 0.088 0.275 
0 1
0 2
1 3
0 4
9 8
0.742 0.320 0.303 0.777 0.913 0.483 0.979 0.096 0.741 
0 1
0 2
2 3
3 4
1 5
3 6
6 7
3 8
9 8
0.148 0.992 0.198 0.827 0.999 0.851 0.507 0.326 0.064 
0 1
1 2
0 3
2 4
1 5
3 6
1 7
6 8
6 5
0.087 0.262 0.611 0.192 0.169 0.781 
0 1
0 2
1 3
2 4
4 5
6 5
0.123 0.224 0.873 0.529 0.255 0.828 
0 1
0 2
2 3
1 4
4 5
7 6
0.627 0.436 0.412 0.890 0.418 0.180 0.251 
0 1
0 2
0 3
1 4
2 5
0 6
7 6
0.673 0.930 0.565 0.583 0.055 0.954 0.344 
0 1
0 2
0 3
3 4
3 5
1 6
8 7
0.117 0.217 0.471 0.554 0.948 0.200 0.186 0.128 
0 1
0 2
2 3
1 4
3 5
1 6
2 7
7 6
0.235 0.039 0.920 0.956 0.069 0.502 0.210 
0 1
1 2
0 3
1 4
2 5
5 6
9 8
0.771 0.356 0.199 0.794 0.943 0.466 0.503 0.524 0.192 
0 1
0 2
1 3
3 4
0 5
5 6
5 7
6 8
5 4
0.082 0.323 0.711 0.993 0.160 
0 1
0 2
2 3
3 4
10 9
0.655 0.320 0.663 0.943 0.307 0.031 0.065 0.439 0.317 0.798 
0 1
0 2
0 3
2 4
2 5
0 6
5 7
2 8
7 9
