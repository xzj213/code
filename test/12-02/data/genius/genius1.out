2 0 1
2 4 6
2 0 3
2 0 6
2 1 4
2 0 1
2 0 2
2 0 1
2 0 1
2 0 1
2 2 3
2 2 4
2 0 1
2 1 2
0
2 3 4
2 0 1
0
2 0 2
2 0 1
2 0 1
0
2 2 3
2 0 1
2 0 1
2 1 4
2 0 3
2 0 1
2 2 3
2 0 2
